README.md

```diff
@@ -51,55 +50,32 @@ functionality; example code for a certificate transparency log and for a
 [log-derived map](#verifiable-log-derived-map) are included to help with this.
 
 
-Working with the Code
----------------------
+Using the Code
+--------------
 
 **WARNING**: The Trillian codebase is still under development, and is not yet
 suitable for production use.  Everything here is subject to change without
 notice &ndash; including APIs, database schemas, and code layout.
 
-### Requirements
-
-You must have Go 1.7.1 installed, and [MySQL](https://www.mysql.com/) or
-[MariaDB](https://mariadb.org/) is required to provide the data storage layer.
+To build and run the Trillian code you need:
 
-Other dependency requirements are then handled by the Go tools (i.e. with `go
-get -d -v -t ./...`).
+ - Go 1.7.1 or later.
+ - [MySQL](https://www.mysql.com/) or [MariaDB](https://mariadb.org/) to provide
+   the data storage layer; see the [MySQL Setup](#mysql_setup) section.
 
-### Building
+Then use the standard Go tools to install other dependencies, build and run unit
+tests:
 
 ```console
```

Comment:
nit: I don't think `console` is valid here. I'd rather be specific; possibly use `bash` instead.

Ditto for others.